app/backend/lib/gis-upload.ts

```diff
@@ -92,12 +96,19 @@ gisUpload.post('/api/analyst/gis', limiter, async (req, res) => {
   }
    
   // time to persist in DB
-  await performQuery(saveGisDataMutation, {input: data}, req)
+  const result = await performQuery(saveGisDataMutation, {input: data}, req)
   .catch((e) => {
     return res.status(400).json({ error: e }).end();
   });
+  
+  const newRow = JSON.parse(JSON.stringify(result));
```

Comment:
Typescript throws error:
```
Property 'data' does not exist on type 'Response<any, Record<string, any>, number> | ExecutionResult<{ [key: string]: any; }, { [key: string]: any; }>'.

Property 'data' does not exist on type 'Response<any, Record<string, any>, number>'.ts(2339)
```
I guess its' because it's result from mutation, not query.